supervised/model_framework.py

```diff
@@ -568,6 +569,13 @@ def load(results_path, model_subpath, lazy_load=True):
         logger.info(f"Loading model framework from {model_path}")
 
         json_desc = json.load(open(os.path.join(model_path, "framework.json")))
+
+        joblib_version_computer = joblib.__version__
+        joblib_version_framework = json_desc.get("joblib_version")
+
+        if joblib_version_framework is not None and joblib_version_computer != joblib_version_framework:
+            logger.warning(f"Joblib version mismatch. Computer: {joblib_version_computer}, Framework: {joblib_version_framework}")
```

Comment:
only warning here is displayed, please raise exception. There is AutoMLException class available https://github.com/mljar/mljar-supervised/blob/master/supervised/exceptions.py